File: Tema 04 - Testing/PetShelter/PetShelter.DataAccessLayer/Models/Person.cs
namespace PetShelter.DataAccessLayer.Models;

public class Person: IEntity
{
    public int Id { get; set; }
    public string Name { get; set; }

    /// <summary>
    ///     Can be used to check if the person is adult or not
    /// </summary>
    public DateTime? DateOfBirth { get; set; }

    public string IdNumber { get; set; }

    public ICollection<Pet> RescuedPets { get; set; }
    public ICollection<Pet> AdoptedPets { get; set; }
    public ICollection<Donation> Donations { get; set; }
}

File: Tema 06 - React/petshelter.ui/src/Models/Person.ts
export class Person {
    idNumber: string;
    name: string;
    //contactNumber: string;
    dateOfBirth:Date

    constructor( name: string,idNumber: string,dateOfBirth:Date) {
        this.idNumber = idNumber;
        this.name = name;
        //this.contactNumber = contactNumber;
        this.dateOfBirth=dateOfBirth;
    }
}